src/main/java/appeng/block/misc/BlockSkyCompass.java

```diff
@@ -164,4 +186,31 @@ public void addCollidingBlockToList( final World w, final BlockPos pos, final Ax
 	{
 
 	}
+
+	@Override
+	public EnumBlockRenderType getRenderType( IBlockState state )
+	{
+		return EnumBlockRenderType.ENTITYBLOCK_ANIMATED;
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public TileEntitySpecialRenderer<? extends AEBaseTile> getTESR()
+	{
+		return new SkyCompassTESR();
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public ItemMeshDefinition getItemMeshDefinition()
+	{
+		return is -> new ModelResourceLocation( "appliedenergistics2:tile.BlockSkyCompass", "normal" );
```

Comment:
I agree with both of you.

Personally, I can only speak about Windows. Here is some food for thought:
- Mojang ships Java 8 with Minecraft when you download it for Windows from them
- Oracle doesn't support Java 7 with security fixes anymore. Only Java 8 is supported
- If you go to java.com to download Java, you'll get Java 8 (and you have been for quite a while)
- The Curse Client seems to default to Java 8 (only did a quick googling, no detailed analysis)

So all in all, I don't even think you'd exclude a significant number of people when you require Java 8. I am all for it!